test/integration/targets/aws_acm/tasks/main.yml

```diff
@@ -0,0 +1,457 @@
+- name: AWS ACM integration test
+  block:
+
+  - set_fact:
+    aws_connection_info: &aws_connection_info
```

Comment:
Needs indenting so that aws_connection_info is  a parameter to set_fact

Reply:
Ah, that's embarrassing. (It's due to a recent move of the `set_fact` task into the block)

Reply:
I've fixed it. (I'll push the commit in a few minutes)